app/javascript/app/service-provider-form.js

```diff
@@ -10,17 +10,25 @@ $(function(){
 
   var toggle_form_fields = function(id_protocol){
     switch(id_protocol){
```

Comment:
There's a similar switch in the function below this one (`issuer_protocol_name`). Should that be updated? Or removed? I can't see that it's even being used based, per the note at the top of the file about `update_issuer` being commented out.

Reply:
Good catch! I think that just updates the example issuer in the help text but that would also be impacted. I'll update, thanks!

Reply:
I updated but then looking further it looks like this JS is actually unused. I think it makes sense to update the `switch` anyway just in case someone turns it on but we can remove it in a larger redesign/refactor, if that makes sense to you. Let me know what you think!

Reply:
> I think it makes sense to update the `switch` anyway just in case someone turns it on but we can remove it in a larger redesign/refactor, if that makes sense to you. Let me know what you think!

Normally I'd be in favor of removing the code so we don't have to deal with situations like this where we're maintaining code that isn't being run anyways. We can always recall it from git history if we need it.

I don't feel strongly though.

Reply:
hm fair enough - I'll do a quick double-check to make sure it's not being referenced and just delete it.

Reply:
all set!